### ISSUE TITLE
Shelly 2PM Gen3

### ISSUE THREAD

**Issue by discostur** [alexryd/homebridge-shelly-ng#140]
Hi,

would be nice to see support for Shelly 2PM Gen3 devices. I already tried https://github.com/cubi1337/homebridge-shelly-ds9 but it seems that the 2PM Gen3 is not yet compatible: 

```
[Shelly NG] [shelly2pmg3-XXX] Unknown device of model "S3SW-002P16EU" discovered.
``` 

Thanks,
Regards
Kilian

**Comment by zsozso-1982** [comment]
> Hi,
> 
> 
> 
> would be nice to see support for Shelly 2PM Gen3 devices. I already tried https://github.com/cubi1337/homebridge-shelly-ds9 but it seems that the 2PM Gen3 is not yet compatible: 
> 
> 
> 
> ```
> 
> [Shelly NG] [shelly2pmg3-XXX] Unknown device of model "S3SW-002P16EU" discovered.
> 
> ``` 
> 
> 
> 
> Thanks,
> 
> Regards
> 
> Kilian

+1! Please make it compatible! Thanky you!

**Comment by crossic84** [comment]
Yeah it would be great to have it in primary package. I would also appreciate if I can get immediate notification when Window Cover starts to open not only once it is fully opened...is there a quick way how can I set it up?

**Comment by ricardopereira** [comment]
I found out that Shelly has official guidance for adding Shelly 2PM Gen3 to Apple Home via Matter so no need to have it on homebridge. You just need to enable Matter on the device (Shelly’s has this page with steps: https://support.shelly.cloud/en/support/solutions/articles/103000343814-connecting-shelly-matter-compatible-devices-to-apple-homekit-application)
